File: fluoxetine_backstage/src/main/java/cn/fluoxetine/fluoxetinebackstage/service/SubjectQuestionService.java
package cn.fluoxetine.fluoxetinebackstage.service;

import cn.fluoxetine.fluoxetinebackstage.pojo.FtbSubjectQuestion;
import cn.fluoxetine.fluoxetinebackstage.pojo.PageResult;

/**
 * @author 11699
 * @date 2020/2/9 - 16:54
 */
public interface SubjectQuestionService {

    /**
     * 模糊查询+分页+查询
     * @param subjectQuestion
     * @param pageNum
     * @param pageSize
     * @return
     */
    public PageResult search(FtbSubjectQuestion subjectQuestion,int pageNum,int pageSize);

    /**
     * 添加
     * @param subjectQuestion
     */
    public void add(FtbSubjectQuestion subjectQuestion);

    /**
     * 删除
     * @param ids
     */
    public void delete(Integer[] ids);

    /**
     * 查找
     * @param id
     * @return
     */
    public FtbSubjectQuestion findOne(Integer id);

    /**
     * 修改
     * @param subjectQuestion
     */
    public void update(FtbSubjectQuestion subjectQuestion);
}

File: fluoxetine_backstage/src/main/resources/static/assets/js/service/subjectQuestionService.js
app.service('subjectQuestionService',function ($http) {

    //模糊查询+分页+查找
    this.search = function (entity,page,size) {
      return $http.post('/subjectQuestion/search?page='+page+'&size='+size,entity);
    };

    //添加
    this.add = function (entity) {
        return $http.post('/subjectQuestion/add',entity);
    };

    //删除
    this.delete = function (ids) {
      return $http.get('/subjectQuestion/delete?ids='+ids);
    };

    //查找
    this.findOne = function (id) {
        return $http.get('/subjectQuestion/findOne?id='+id);
    };

    //修改
    this.update = function (entity) {
        return $http.post('/subjectQuestion/update',entity);
    }

});